src/fibad/verbs/umap.py

```diff
@@ -64,7 +82,9 @@ def run(self, input_dir: Optional[Union[Path, str]] = None, **kwargs):
         sample_size = np.min([config_sample_size if config_sample_size else np.inf, total_length])
         rng = np.random.default_rng()
         index_choices = rng.choice(np.arange(total_length), size=sample_size, replace=False)
-        data_sample = inference_results[index_choices].numpy()
+
+        # If the input to umap is not of the shape [samples,input_dims] we reshape the input accordingly
+        data_sample = inference_results[index_choices].numpy().reshape(sample_size, -1)
```

Comment:
This took me a little while to fully understand what was going on here. The `-1` syntax was unfamiliar, perhaps an update to this comment could be:
`# Reshape the input data to be an array of 1d arrays`

Perhaps this is very well know numpy syntax, and if so, feel free to ignore, but would the following suggestion work? It seems a bit more intuitive to me to specify the shape as a tuple. Similar story for line 107. 
```suggestion
        data_sample = inference_results[index_choices].numpy().reshape((sample_size, -1))
```